lib/mixins/validatable-http-message.js

```diff
@@ -428,13 +429,16 @@ but body is not a parseable JSON:\n${error.message}\
     if (!contentType) {
       return false;
     }
-
-    const parsed = mediaTyper.parse(`${contentType}`);
-    const isJson =
-      (parsed.type === 'application' && parsed.subtype === 'json') ||
-      parsed.suffix === 'json';
-
-    return isJson;
+    try {
+      const { type } = contentTypeModule.parse(`${contentType}`);
+      const parsed = mediaTyper.parse(type);
+      return (
+        (parsed.type === 'application' && parsed.subtype === 'json')
+        || parsed.suffix === 'json'
+      );
+    } catch (e) {
```

Comment:
We shouldn't silence an error. Can we log it to console?

Reply:
The error gets thrown in case the value is not a valid media type. The only reponsibility of the function is to assess whether the value is JSON-related or not, so I think this is a valid approach.

The `contentType` value is basically a user input, it can be any kind of rubbish, and it is not Gavel's problem if it's invalid media type as far as it matches real/expected response.

Reply:
I added an explanatory comment in 519965b so it doesn't trigger a red flag when someone reads it